src/shared/modules/discovery/discoveryDuck.js

```diff
@@ -156,7 +167,11 @@ export const discoveryOnStartupEpic = (some$, store) => {
               getAllowedBoltSchemes(store.getState()),
               host
             )
-            store.dispatch(updateDiscoveryConnection({ host })) // Update discovery host in redux
+            const supportsMultiDb =
+              parseInt((result.neo4j_version || '0').charAt(0)) >= 4
```

Comment:
I'm thinking there's a better way to get version number? Or is this fine?

Reply:
So do you suggest we check that the version starts with 4 and is not running on aura?

Reply:
Also, good catch!

Reply:
Some aura editions do support multi-db, so it's tricky. I suggest you talk to Aura and ask which editions that has multi-db support and what version string those editions expose in the discovery endpoint.

Reply:
Asked just now!

Reply:
It seems aura only supports the system + 1 other database at this moment. I think it'd make sense to disable the field for aura, WDYT?